### DISCUSSION TITLE
Display a countdown timer

### DISCUSSION THREAD

**Discussion by ThatTallGuy21** [piitaya/lovelace-mushroom#723] [Ideas]
Currently an entity card is the only way that you can display a countdown timer. It would be great to be able to display the remaining time of an active timer in a template card or a new card.

**Comment by flemmingss** [comment]
Agree, and I need it to. As you might have read I have posted about this timer issue here: https://community.home-assistant.io/t/implement-a-realtime-remaining-attributt-to-timer/462744
It works in timer-bar-card, but I am looking for a way to implement it as secondary info in a mushroom template card.
https://github.com/rianadon/timer-bar-card
Don't need the bar, just countdown

**Comment by RickeyWard** [comment]
See: https://github.com/piitaya/lovelace-mushroom/issues/684

![comparison](https://user-images.githubusercontent.com/342468/229946657-78fd9659-b6d8-4946-926a-afedfadfd359.gif)
